### PR TITLE
Adding Keplergl feedstock

### PR DESCRIPTION
**Pull request by lixun910**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by lixun910** [comment]
@conda-forge/help-python The recipe has passed all checks and is ready for your review. Thank you!

File: recipes/keplergl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cb21047b2104413af1c00ef1ac75794a19e0b578e51c69c86713911d97370167
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
```
Comment (ocefpaf) [inline recipes/keplergl/meta.yaml:13]: Please remove the comments from the example.
```suggestion
```

File: recipes/keplergl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  imports:
+    - keplergl
+
+about:
+  home: https://github.com/keplergl/kepler.gl
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Kepler.gl for Jupyter'
+  # The remaining entries in this section are optional, but recommended.
```
Comment (ocefpaf) [inline recipes/keplergl/meta.yaml:46]: Please remove the comments from the example.
```suggestion
```

File: recipes/keplergl/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    - jupyter nbextension install --py --sys-prefix keplergl
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools
+    - pip
+    - jupyter
+    - pandas
+    - geopandas
```
Comment (ocefpaf) [inline recipes/keplergl/meta.yaml:26]: You probably don't need `jupyter`, `pandas`, and `geopandas` at host. Unless the package imports itself at build time.

Comment (lixun910) [reply recipes/keplergl/meta.yaml:26]: Seems that jupyter pandas geopandas are needed at host. Thanks!

Comment (ocefpaf) [reply recipes/keplergl/meta.yaml:26]: Sorry for the wild goose chase!

Comment (lixun910) [reply recipes/keplergl/meta.yaml:26]: Not a problem😁 Thank you very much!

**Pull request merged**